plugins/source/postgresql/resources/plugin/plugin_test.go

```diff
@@ -52,11 +52,12 @@ func getTestConnectionString() string {
 	return testConn
 }
 
-func create_test_table(ctx context.Context, c *client.Client, table *schema.Table) error {
+func createTestTable(ctx context.Context, conn *pgxpool.Pool, table *schema.Table) error {
```

Comment:
Most of these test changes were made so that we can pass in a separate connection (not the one managed by the plugin). I think it's better this way, but it's also because now the plugin will error with "pattern matching no tables" if a table is specified that doesn't exist yet